### PR TITLE
build docker image with static UI

### PR DESCRIPTION
**Pull request by AndrewChubatiuk**
## What does this PR change?
Added simple docker image for UI
Added simple CI build for docker image

## How was this PR tested?
Was tested in my EKS cluster

### CODE REVIEW

File: ui/src/Reports.js [hunk 1]
```diff
@@ -154,12 +154,6 @@ const ReportsPage = () => {
       if (resp.data && resp.data.length > 0) {
         const allocationRange = resp.data
         for (const i in allocationRange) {
-          // update cluster aggregations to use clusterName/clusterId names
```
Comment (AjayTripathy) [inline ui/src/Reports.js:157]: Curious what this removal is for?

Comment (AndrewChubatiuk) [reply ui/src/Reports.js:157]: Tested it on setup with single victoria metrics cluster, where we aggregate metrics from multiple EKS clusters. We do not want to setup prometheus on each EKS as it's describe in doc. Reverted these changes back

Comment (AndrewChubatiuk) [reply ui/src/Reports.js:157]: @AjayTripathy all builds have passed

**Review by AjayTripathy — APPROVED** [review]

**Comment by AjayTripathy** [comment]
Thanks @AndrewChubatiuk !

File: ui/default.nginx.conf [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+    add_header Cache-Control "max-age=300";
+    location / {
+        try_files $uri $uri/ /index.html;
+    }
+
+    add_header ETag "1.96.0";
+    listen 9090;
+    listen [::]:9090;
+    resolver kube-dns.kube-system.svc.cluster.local valid=5s;
```
Comment (a7i) [inline ui/default.nginx.conf:54]: Is there a reason this way added? kube-dns is long gone

File: ui/default.nginx.conf [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    location /healthz {
+        return 200 'OK';
+    }
+    location /allocation {
+        proxy_connect_timeout       180;
+        proxy_send_timeout          180;
+        proxy_read_timeout          180;
+        set $server http://cost-analyzer.kubecost.svc.cluster.local:9003;
```
Comment (a7i) [inline ui/default.nginx.conf:62]: why is this not set to opencost? but rather kubecost?

File: ui/default.nginx.conf (hunk 2 above)
Comment (AjayTripathy) [reply ui/default.nginx.conf:54]: whoops. Good catch, thank you. Let's get rid of this.

**Pull request merged**